sdk/dnsresolver/Azure.ResourceManager.DnsResolver/tests/Tests/DnsResolverTests.cs

```diff
@@ -35,17 +38,13 @@ public async Task CreateDnsResolverAsync()
             // ARRANGE
             var dnsResolverName = Recording.GenerateAssetName("dnsResolver-");
             var vnetName = Recording.GenerateAssetName("vnet-");
-            var vnetId = $"/subscriptions/{TestEnvironment.SubscriptionId}/resourceGroups/{TestEnvironment.ResourceGroup}/providers/Microsoft.Network/virtualNetworks/{vnetName}";
+            await CreateDnsResolverCollectionAsync();
+            var vnetId = DefaultVnetID.ToString();
```

Comment:
This line looks redundant - it is already a string

Reply:
fixed